flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/master";
```

Comment:
This is missing `flake-utils`. Like this the lockfile output will never be updated with `nix flake update`.